projects/control-service/cicd/.gitlab-ci.yml

```diff
@@ -339,4 +322,5 @@ control_service_vdk_heartbeat_release:
   only:
     refs:
       - main
-    changes: *control_service_change_locations
```

Comment:
Why did you not just change control_service_change_locations to be the union of control_service_code_change_locations and control_service_helm_change_locations?

Reply:
Do you know how ?

From what I've found Merging arrays is not really supported in YAML https://github.com/yaml/yaml/issues/35 
Nor in gitlab references .

Reply:
Can changes take a list then instead of declaring the property twice?

Reply:
I changed it to use rules instead

Reply:
now there is lots of duplication. 
if there is no way of getting rid of the duplication then I'm still happy for it to be merged.

Reply:
I will do it in subsequent PR. I want to make sure the CI is working functionally correct. Then I'd refactor it.

Reply:
Cool